person.rb

```diff
@@ -1,15 +1,17 @@
 require_relative 'nameable'
+require_relative 'rental'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Getters and Setters for @rentals should be present. Please remove `:rentals` from `attr_reader` and set it to `attr_accessor`